src/guide/kuzzle-backend-setup/aws-marketplace.md

```diff
@@ -4,4 +4,68 @@ algolia: true
 title: AWS Marketplace
 description: AWS Marketplace
 order: 1200
----
\ No newline at end of file
+---
+
+# AWS Marketplace
+> ⚠️ To follow this tutorial, you need a __valid AWS account__.
+
+_In this guide, you'll learn where and how to use our AWS Marketplace AMI. It's a
+ good way to test Kuzzle in a cloud environment.In addtion, we recommend you to
+ use our [Kuzzle Admin Console](http://console.kuzzle.io). It's the most comfy
+ way to play with Kuzzle._
+
+## Get the AMI
+
+Our AMI is stored on AWS Marketplace. It's setup with:
+  * Ubuntu (__16.04__)
+  * Kuzzle (__latest__) with MQTT protocol support.
+  * Elasticsearch (__v5.4.1__).
+  * Redis (__v3.2.12__).
+
+Go to the marketplace and type __kuzzle__ in search form.
+You should see one result:
+
+  ![Search result image]()
+
+Choose your Amazon EC2 instance type (minimal instance type is __t2-medium__).
+
+Recover the public IP or the hostname provided by AWS before continue.
+Check if Kuzzle is up and running simply perform an HTTP request:
+
+```sh
+$ curl http://yourInstanceIpOrHostname:7512/_now\?pretty
+{
+  "requestId": "9abc8b22-d033-4579-a2b8-35b694839ee1",
+  "status": 200,
+  "error": null,
+  "controller": "server",
+  "action": "now",
+  "collection": null,
+  "index": null,
+  "volatile": null,
+  "result": {
+    "now": 1531127663204
+  }
+}
+```
+
+You should see the current UNIX formatted timestamp.
+If not, wait a few minutes and retry previous request.
+
+## Connect and create first user
+
+Go to our [Kuzzle Admin Console](http://console.kuzzle.io), type your Kuzzle instance
```

Comment:
"Open the Kuzzle Admin Console and fill the form with the address of your Kuzzle instance. Kuzzle will then ask you to create your first user (we recommend to reset the rights for the anonymous user). At this point, you'll be able to log-in."